lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlanarSurface.rb

```diff
@@ -54,8 +54,10 @@ def planar_surface_apply_standard_construction(planar_surface, climate_zone, pre
         cons_set = space.model.building.get.defaultConstructionSet.get
         construction = get_default_surface_cons_from_surface_type(surface_category, surface_type, cons_set)
       end
-
-      return previous_construction_map if construction.nil?
+      prm_raise(construction,
```

Comment:
Do we need to add the conditional to only call prm_raise if construction.nil? is true?

Reply:
@dmaddoxwhite  that is handled inside the `prm_raise` function
See: https://github.com/NREL/openstudio-standards/blob/0fe207df7d68d3a4140232320a7a275d2ae452f8/lib/openstudio-standards/utilities/assertion.rb#L12

Reply:
I see, so if construction is nil, then it is interpreted as "FALSE" by prm_raise.

Reply:
@dmaddoxwhite That is correct - in ruby, False and nil is Falsey.
Alternatively, you can also do:
```ruby
prm_raise(construction == nil, @sizing_run_dir, ...)
```
This would explicitly expose the condition logic - I am OK with both way in this specific case.